study.md

```diff
@@ -160,15 +170,27 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+
+A response is what is sent back from the server in "response" to an HTTP request from the client (e.g. browser).
+Includes: Response Status Code, Content Type, Last Modified Date, Server details etc.
+
+200 OK - Request is successful
+
+201 Created - Confirms success of a POST or PUT request. Let's you know the resource was created/updated
+
+400 Bad Request - Issue with the request. Happens if data is in the wrong format or doesn't pass validation (common for POS/PUT requests)
+
+404 Not Found - Resource could not be located. Occurs if a URL has no associated resources.
+
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl --request https://www.reddit.com
+(although, this results in a message indicating that reddit thinks I'm a bot and only one request/second can be sent. I only sent the one, so not sure what that means..)
```

Comment:
good you typically include the type of request `GET`, however if you don't specify I think it defaults to a GET request.